### PR TITLE
Account for commas when encoding CSVs

### PR DESCRIPTION
**Pull request by itsmontoya**

### CODE REVIEW

File: encoder_test.go [hunk 1]
```diff
@@ -18,6 +18,10 @@ func TestEncoder(t *testing.T) {
 		Age:       32,
 		City:      "Portland",
 		State:     "Oregon",
+		Notes: `"Favorite foods:
```
[the remark this answers is not in the document]
Comment (kole) [reply encoder_test.go]: Ideally PR's only contain changes related to the current PR. Can we back this out and put in a new PR so we aren't mixing concerns?

File: encoder_test.go [hunk 2]
```diff
@@ -26,6 +30,15 @@ func TestEncoder(t *testing.T) {
 		Age:       30,
 		City:      "Portland",
 		State:     "Oregon",
+		Notes:     "foo bar",
```
[the remark this answers is not in the document]
Comment (kole) [reply encoder_test.go]: Ideally PR's only contain changes related to the current PR. Can we back this out and put in a new PR so we aren't mixing concerns?

File: encoder_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (itsmontoya) [reply encoder_test.go]: Moved these changes to another PR, then changed the base of this PR to https://github.com/hatchify/csvhelper/pull/9

File: encoder_test.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (itsmontoya) [reply encoder_test.go]: Moved these changes to another PR, then changed the base of this PR to https://github.com/hatchify/csvhelper/pull/9

**Comment by kole** [comment]
@itsmontoya since I'm not familiar with the project, I do have a couple questions about some things.
1. There don't appear to be any build processes built-in to this repo... can we confirm that the tests pass and the project builds?
2. Does this pass linting?

**Comment by itsmontoya** [comment]
1. Since this is a library, there aren't any build scripts and/or processes. We do have tests built into go, those tests can be called by running `go test` (if go is installed on your workstation). Test suite definitely passes though, I can provide screenshots in the interim (while go is being setup on your machine)
2. Yes, the go community is very strict on linting and formatting :)

**Comment by kole** [comment]
Ok makes sense. I think this one LGTM — although I hesitate a little since I don't know go well.

**Comment by itsmontoya** [comment]
Your comments are highly appreciated, thank you!

**Pull request merged**